log/log.go

```diff
@@ -0,0 +1,47 @@
+package log
+
+import (
+	"io/ioutil"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var log *logrus.Logger
+
+const DebugLevel = logrus.DebugLevel
+
+func Debugf(format string, args ...interface{}) {
+	log.Debugf(format, args...)
```

Comment:
I believe because of the extra layer, your wrapper will log incorrect lines, you should fix that.
 Also I would like to avoid using global variables here.